.github/workflows/pull_request.workflow.yml

```
@@ -8,21 +8,21 @@ jobs:
     runs-on: ubuntu-18.04
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
+      - uses: actions/setup-node@v2
         with:
-          node-version: 12
+          node-version: 14
       - uses: ./.github/actions/eslint
 
   unit-tests:
     name: Unit Tests - Node.js v${{ matrix.node-version }}
     needs: [lint]
     strategy:
       matrix:
-        node-version: ['12.16.3', '12.20.0']
+        node-version: ["16", "16"]
```

I think we should test those version
```suggestion
        node-version: ["14", "16"]
```